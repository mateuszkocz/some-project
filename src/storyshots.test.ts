/**
 * This file initializes the StoryShots functionality.
 * Read more: https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core
 */

import initStoryshots from "@storybook/addon-storyshots"

initStoryshots({
  /* configuration options */
})
