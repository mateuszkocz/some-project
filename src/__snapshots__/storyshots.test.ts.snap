// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Layout|Dashboard/Header default 1`] = `
<header
  style={
    Object {
      "background": "rebeccapurple",
      "marginBottom": "1.45rem",
    }
  }
>
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "1.45rem 1.0875rem",
      }
    }
  >
    <h1
      style={
        Object {
          "margin": 0,
        }
      }
    >
      <a
        href="/"
        style={
          Object {
            "color": "white",
            "textDecoration": "none",
          }
        }
      >
        Money Manager
      </a>
    </h1>
  </div>
</header>
`;

exports[`Storyshots Layout|Dashboard/Header new 1`] = `
<header
  style={
    Object {
      "background": "rebeccapurple",
      "marginBottom": "1.45rem",
    }
  }
>
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "1.45rem 1.0875rem",
      }
    }
  >
    <h1
      style={
        Object {
          "margin": 0,
        }
      }
    >
      <a
        href="/"
        style={
          Object {
            "color": "white",
            "textDecoration": "none",
          }
        }
      >
        Money Manager New
      </a>
    </h1>
  </div>
</header>
`;
